ui/src/react-admin/modules/content-page/views/ContentPageDetailMetaData.tsx

```diff
@@ -12,6 +8,9 @@ import {
 	TagOption,
 	Thumbnail,
 } from '@viaa/avo2-components';
+import { compact, get } from 'lodash-es';
+import moment from 'moment';
+import React, { FunctionComponent } from 'react';
```

Comment:
Doesn't this throw linting warnings about import groups?

Reply:
no for me it doesn't
might have something to do with the monorepo and split version of the .eslint config file

I'll look at it some other time.